ams/server/src/main/java/com/netease/arctic/server/persistence/mapper/OptimizingMapper.java

```diff
@@ -112,12 +112,12 @@ List<OptimizingProcessMeta> selectOptimizingProcesses(
         + "end_time, status, fail_reason, optimizer_token, thread_id, rewrite_output, metrics_summary, properties) "
         + "VALUES ",
     "<foreach collection='taskRuntimes' item='taskRuntime' index='index' separator=','>",
-    "(#{taskRuntime.taskId.processId}, #{taskRuntime.taskId.taskId}, #{taskRuntime.retry},"
+    "(#{taskRuntime.taskId.processId}, #{taskRuntime.taskId.taskId}, #{taskRuntime.runTimes},"
```

Comment:
Do you think we should change the column name in the database table too?

Reply:
I think we could do this next time.

The logic before is not completed and causes consistency problem. Right now even we don't change column name here, the only consequence is that the true retry num would be one less.

If we change column name here, the upgrade would be complicated which is not necessary for now

Reply:
Looks fine to me.